src/Resources/views/standard_layout.html.twig

```diff
@@ -103,10 +103,10 @@ file that was distributed with this source code.
                 {%- endif -%}"
                 data-sonata-admin='{{ {
                     config: {
-                        CONFIRM_EXIT: sonata_admin.adminPool.getOption('confirm_exit') ? 'true' : 'false',
-                        USE_SELECT2: sonata_admin.adminPool.getOption('use_select2') ? 'true' : 'false',
-                        USE_ICHECK: sonata_admin.adminPool.getOption('use_icheck') ? 'true' : 'false',
-                        USE_STICKYFORMS: sonata_admin.adminPool.getOption('use_stickyforms') ? 'true' : 'false'
+                        CONFIRM_EXIT: sonata_admin.adminPool.getOption('confirm_exit') ? true : false,
+                        USE_SELECT2: sonata_admin.adminPool.getOption('use_select2') ? true : false,
+                        USE_ICHECK: sonata_admin.adminPool.getOption('use_icheck') ? true : false,
+                        USE_STICKYFORMS: sonata_admin.adminPool.getOption('use_stickyforms') ? true : false
```

Comment:
If you're going to do this, please remove `? true : false` entirely, and cast to boolean if really needed (I don't know if it is)

Reply:
but less readable :(

Reply:
>  if really needed (I don't know if it is)
Please answer this first

Reply:
Just checked — looks like those options are coming as boolean already, so no casting is needed — thanks for catching this.